koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -339,7 +351,7 @@ def _get_export_time_for_big_query(self, scan_start, scan_end, key):
         """
         Logic to set export_time in the manifest.
         """
-        dh = DateHelper()
+        today = DateAccessor().today().date() + relativedelta(days=1)
```

Comment:
should we call this tomorrow?

Reply:
bahaha yes. I added a day here because I was getting one day of duplicate data during the day to day flow. It worked and then I just moved on to the next thing. 😂 Whoops